website/docs/guides/orchestration/custom-cicd-pipelines/1-creating-custom-cicd-pipelines.md

```diff
@@ -0,0 +1,431 @@
+
+Here’s a video showing the steps as well:
+
+<WistiaVideo id="iub17te9ir" />
+
+
+
+### 2. Put your dbt Cloud API key into your repo
+
+This next part will happen in you code hosting platform. We need to save your API key from above into a repository secret so the job we create can access it. It is **not** recommended to ever save passwords or API keys in your code, so this step ensures that your key stays secure, but is still usable for your pipelines. 
+
+<details>
+<summary> GitHub </summary>
+    
+In GitHub:
+
+- Open up your repository where you want to run the pipeline (the same one that houses your dbt project)
+- Click *Settings* to open up the repository options
+- On the left click the *Security* dropdown
+- From that list, click on *Actions*
+- Towards the middle of the screen, click the *New repository secret* button
+- It will ask you for a name, so let’s call ours `DBT_API_KEY`
+    - **It’s very important that you copy/paste this name exactly because it’s used in the scripts below.**
+- In the *Value* section, paste in the key you copied from dbt Cloud
+- Click *Add secret* and you’re all set!
+
+** A quick note on security: while using a repository secret is the most straightforward way to setup this secret, there are other options available to you in GitHub. They’re beyond the scope of this guide, but could be helpful if you need to create a more secure environment for running actions. Checkout GitHub’s documentation on secrets [here](https://docs.github.com/en/actions/security-guides/encrypted-secrets).*
+
+Here’s a video showing these steps:
+
+<WistiaVideo id="u7mo30puql" />
+
+</details>
+
+<details>
+<summary> GitLab </summary>
+    
+In GitLab:
+
+- Open up your repository where you want to run the pipeline (the same one that houses your dbt project)
+- Click *Settings* > *CI/CD*
+- Under the *Variables* section, click *Expand,* then click *Add variable*
+- It will ask you for a name, so let’s call ours `DBT_API_KEY`
+    - **It’s very important that you copy/paste this name exactly because it’s used in the scripts below.**
+- In the *Value* section, paste in the key you copied from dbt Cloud
+- Make sure the check box next to *Protect variable* is unchecked, and the box next to *Mask variable* is selected (see below)
+    - “Protected” means that the variable is only available in pipelines that run on protected branches or protected tags - that won’t work for us because we want to run this pipeline on multiple branches. “Masked” means that it will be available to your pipeline runner, but will be masked in the logs.
+    
+    ![View of the GitLab window for entering DBT_API_KEY](/img/guides/orchestration/custom-cicd-pipelines/dbt-api-key-gitlab.png)
+    
+    Here’s a video showing these steps:
+    
+    <WistiaVideo id="rgqs14f816" />
+
+</details>
+
+
+
+### 3. Copy down the Python script that calls dbt Cloud
```

Comment:
"Copy down" is a bit vague - maybe "Create script to trigger dbt Cloud job via API call"

Reply:
I like that working much better, will update